Specflow/Language.feature

```diff
@@ -0,0 +1,37 @@
+﻿Feature: Language
+
+As a Tester I would like to add, edit and delete Languages successfully so users can have their languages 
+showing successfull in the portal.
+
+@Positive
+Scenario: Add Language and set level
+	Given I logged into the portal successfull
+	When I navigate to Profile page
+	When I add the Language in the Language section
+	Then The languange shoould be added successfully	
+	
+
+@Extended-Positive
```

Comment:
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests
You all need to pass the values down from feature files to the steps see examples below


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/